syntaxes/template.json

```diff
@@ -32,30 +32,60 @@
     },
 
     "propertyBinding": {
-      "begin": "(\\[.*\\])(=)(\"|')",
+      "begin": "(\\[\\s*@?[-_a-zA-Z0-9.$]*\\s*])(=)([\"'])",
       "beginCaptures": {
         "1": {
-          "name": "entity.other.attribute-name.html"
+          "name": "entity.other.attribute-name.html entity.other.ng-binding-name.html",
```

Comment:
What do you think of making the scope name more specific, like `ng-property-binding-name`?

Reply:
Yeah, that's a good idea now that this scope isn't generic anymore.

Reply:
Updated! 0a254fe6fcd05a2f9fd60473328ed83ddd7d72ef